### PR TITLE
Add support for Oregon Scientific UVR128 UV sensor

### PR DESCRIPTION
**Pull request by ovrheat**
Add support for Oregon Scientific UVR128 UV sensor.

### CODE REVIEW

File: src/devices/oregon_scientific.c [hunk 1]
```diff
@@ -435,6 +436,20 @@ static int oregon_scientific_v2_1_parser(bitbuffer_t *bitbuffer) {
...
       return 1;
+	} else if (sensor_id == ID_UVR128 && num_valid_v2_bits==297) {
+		if ((validate_os_v2_message(msg, 297, num_valid_v2_bits, 12) == 0)) {
+		int uvidx = get_os_uv(msg, sensor_id);
+		data = data_make(
+		  "time",           "",           DATA_STRING, time_str,
+		  "model",          "",     	  DATA_STRING, "Oregon Scientific UVR128",
+		  "id",             "House Code", DATA_INT,    get_os_rollingcode(msg, sensor_id),
+		  "uv",             "UV Index",   DATA_FORMAT, "%u", DATA_INT, uvidx,
+		  "battery",        "Battery",    DATA_STRING, get_os_battery(msg, sensor_id)?"LOW":"OK",
```
Comment (merbanan) [inline src/devices/oregon_scientific.c]: I think we concluded to use battery_low as generic name.

**Comment by merbanan** [comment]
Looks good.

File: src/devices/oregon_scientific.c (hunk 1 above)
Comment (zuckschwerdt) [reply src/devices/oregon_scientific.c]: yes, but it looks like this just a plain copy of all the other cases in the file. I'm not happy with "%u" either (uvidx is 0 to 165). I guess we should later change all of those "battery" instances with one commit.

**Comment by ovrheat** [comment]
Sorry, I'm just asking... It's not clear to me if it's ok, or if is there something I should fix so to have the pull merged (battery? "%u" ?)... 
My next pull request will be date and time decoding for the RTGN318, but I'm waiting for this pull to be closed. Thank you.

**Pull request merged**